checkers/deprecatedComment_checker.go

```diff
@@ -134,6 +142,12 @@ func (c *deprecatedCommentChecker) VisitDocComment(doc *ast.CommentGroup) {
 				return
 			}
 		}
+
+		if strings.HasPrefix(l, DeprecatedPrefix) && prev != nil && *prev != "" {
+			c.warnParagraph(comment)
+			return
+		}
+		prev = &rawLine
```

Comment:
I don't think there is a need for a `*string` for `prev`

This would be enough with `prev` being a `string`

```suggestion
		if strings.HasPrefix(l, DeprecatedPrefix) && prev != "" {
			c.warnParagraph(comment)
			return
		}
		prev = rawLine
```